resources/assets/js/localizations/fr.json

```diff
@@ -18,26 +18,12 @@
   "apl.expandAllSkills": "Élargir les compétences",
```

Comment:
I don't think this file is actually being used by anything anymore, but I'm not confident enough to say remove it

Reply:
Hmm i'm not sure either, but I can find out.

Reply:
Hey, so I was able to remove most of the message data from the `localizations` folder. The remaining messages are needed in the `alert.js` file, which is used for deleting draft job posters in the Manager job index page. There weren't that many messages so I just moved them into the file itself and removed the folder.